Tasks/AzureResourceManagerTemplateDeploymentV3/task.json

```diff
@@ -15,7 +15,7 @@
     "version": {
```

Comment:
please do same fix for ARG v2 task as well

Reply:
Done